homeassistant/components/influxdb.py

```diff
@@ -179,7 +182,7 @@ def influx_event_listener(event):
         for key, value in state.attributes.items():
             if key in tags_attributes:
                 json_body[0]['tags'][key] = value
-            elif key != 'unit_of_measurement':
```

Comment:
Appreciate thoughts on whether it would be better to just remove the conditional (key != 'unit_of_measurement') on line 182 and provide a uom field in all cases? Consistency with other attributes has some merit and there are advantages to having this in a field in that you can use it in Grafana tables etc.

Reply:
I like consistency 👍 better to always include it imo (but I'm not a user of this component)